packages/base/any/kernels/4.19-lts/configs/x86_64-all/x86_64-all.config

```diff
@@ -3311,6 +3311,40 @@ CONFIG_USB_STORAGE=y
 # CONFIG_USB_GPIO_VBUS is not set
 # CONFIG_USB_ISP1301 is not set
 # CONFIG_USB_GADGET is not set
+#
+# USB Peripheral Controller
+#
+# CONFIG_USB_FOTG210_UDC is not set
+# CONFIG_USB_GR_UDC is not set
+# CONFIG_USB_R8A66597 is not set
+# CONFIG_USB_PXA27X is not set
+# CONFIG_USB_MV_UDC is not set
+# CONFIG_USB_MV_U3D is not set
+# CONFIG_USB_M66592 is not set
+# CONFIG_USB_BDC_UDC is not set
+# CONFIG_USB_AMD5536UDC is not set
+# CONFIG_USB_NET2272 is not set
+# CONFIG_USB_NET2280 is not set
+# CONFIG_USB_GOKU is not set
+# CONFIG_USB_EG20T is not set
+# CONFIG_USB_DUMMY_HCD is not set
+CONFIG_USB_LIBCOMPOSITE=y
+CONFIG_USB_F_ACM=y
+CONFIG_USB_U_SERIAL=y
+CONFIG_USB_CONFIGFS=y
+# CONFIG_USB_CONFIGFS_SERIAL is not set
+CONFIG_USB_CONFIGFS_ACM=y
```

Comment:
Why not build these as modules? (Strange the bzImage size did not change in your tests.)

Reply:
Hmn... in my test, I did two times builds. And in every build, I re-pull a clear base and only change this configure file.
I originally thought almost kernel config in ONL would be built-in. Is it requested to change it to module ?

Reply:
Historically you are correct, we minimized kernel modules, but lately with features for single vendors/devices we prefer modules.